ansible/slurm.yml

```diff
@@ -1,21 +1,12 @@
 ---
-- hosts: cluster_login
-  become: yes
-  roles:
-    - role: openhpc_control
-
-# Cluster batch setup
-- hosts: cluster_batch
-  become: yes
-  roles:
-    - role: openhpc_compute
-
-# Cluster setup
-- hosts: 
-  - cluster_login
+- hosts:
+  - cluster_control
```

Comment:
We probably want to include cluster_login here as well, to ensure it gets the OpenHPC runtime environment installed.

Reply:
+1